src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllTransactionDetailsForAccountByDate.sql

```diff
@@ -56,6 +56,59 @@ AND DateCreated >= @FromDate
 AND DateCreated < @ToDate
 
 
+
+-- We will use this View to get payments that relate to transaction lines and are funding source 1,2,3
+;WITH AllFunds (DateCreated, AccountId, Ukprn, Uln, PeriodEnd, PaymentId, FundingSource, Amount, PaymentMetaDataId, TransactionTypeDesc,
+	TrainingProvider,
+	Apprentice,
+	ApprenticeTrainingCourse,
+	ApprenticeTrainingCourseLevel)
+AS
+(
+--	SELECT PaymentId, FundingSource, Amount, p.AccountId, p.Ukprn, p.Uln, p.PeriodEnd, p.PaymentMetaDataId
+	SELECT 
+			transLine.DateCreated,
+			p.AccountId, 
+			p.Ukprn, 
+			p.Uln, 
+			p.PeriodEnd, 
+			PaymentId, 
```

Comment:
Would it be better to qualify all columns in the result list?

Reply:
Done